walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,140 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    if (request == null) {
+      return Arrays.asList(TimeRange.WHOLE_DAY);
```

Comment:
avoid returning mutable collections. If you already have Guava:collect can you use `return ImmutableList.of(TimeRange.WHOLE_DAY)`. Same comment throughout the file.